### ISSUE TITLE
Integrate `event_update_notification` app (per calendar notification settings)

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/calendar#1883]
I use the app event_update_notification and I think it cloud be nice, if the app would be integrated in the calendar app. 

**Describe the solution you'd like**
If a calendar is shared the owner can check a checkbox "send update notifications to shared users"

**Describe alternatives you've considered**
Using the app itself (but choosing a calendar is not possible, when the app is active it sends for all shared calendars)

**Comment by georgehrke** [comment]
What is the benefit of integrating it into the calendar directly? 

cc @nickvergessen

**Comment by tcitworld** [comment]
If I understand correctly, the benefit would be a per calendar notification setting, but that could be done on the event_update_notification app as well.

**Comment by nickvergessen** [comment]
Yeah, we should also wait for the discussion about activities and notifications being merged. That would duplicate the efforts.

**Comment by SebastianKrupinski** [comment]
Closing this in favour of https://github.com/nextcloud/calendar/issues/7599
